src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -75,7 +75,21 @@
     <GenerateResourcesCode
         ResxFilePath="$(StringResourcesPath)" 
         OutputSourceFilePath="$(IntermediateResOutputFileFullPath)" 
-        AssemblyName="$(AssemblyName)" />
+        AssemblyName="$(AssemblyName)" >
+
+        <Output TaskParameter="NormalizedAssemblyName" PropertyName="_NormalizedAssemblyName" />
```

Comment:
That is a good idea too. we may consider have both, the override property and the normalization too.